tests/projects/test_projects_cli.py

```diff
@@ -1,25 +1,29 @@
 import pytest
 
-import mlflow
 from mlflow import cli
-from mlflow.utils.file_utils import TempDir
-from tests.integration.utils import invoke_cli_runner, update_temp_env
-from tests.projects.utils import TEST_PROJECT_DIR, GIT_PROJECT_URI
+from tests.integration.utils import invoke_cli_runner
```

Comment:
Refactoring the existing tests to use mock helpers from elsewhere + adding new integration tests for SSH URIs